### DISCUSSION TITLE
Potential issue with usb tethering shutting down

### DISCUSSION THREAD

**Discussion by Gab20122** [Mygod/VPNHotspot#579] [Q&A]
First, thank you for such a great app! It’s worked amazing 99% of the time for my use 😊

I run my moto edge (2022) rooted tethered via usb using your app on a gl.inet MT3000 travel router (I need Ethernet and it supports greater range & more clients vs just using hotspot on the phone)

When I leave the phone tethered for an extended period of time, the toggle for the vpn tunnel “rndis0” on the app switches off and all data gets routed as regular hotspot. I have to kill the usb tether and re-enable the “rndis0” toggle to get things back working.

any way to see what’s causing this? I’m 100% new to your app and could be making a mistake on my end.
Thank you!

**Comment by Mygod** [comment]
Try disabling battery saving features on this app.
